README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Overview
 
-This is a Kubernetes cluster that includes logging, monitoring, and operational
+This is a Kubernetes 1.4.0 cluster that includes logging, monitoring, and operational
```

Comment:
We should not put the version in here for the burden of having to update that for all the releases we are going to make